### ISSUE TITLE
Update Unit Leader data

### ISSUE THREAD

**Issue by justdave** [justdave/nslodge-wp-customizations#18]
We have a current list of units from the council in the database now thanks mostly to Jarret, but we didn't manage to import the list of unit leaders that goes with it.  It needs some manual intervention because a lot of them already exist in the database and it'll cause duplicates if we import the spreadsheet as-is. So we have to go through the list, manually update units and contact info on people who are already in the database, then delete them from the spreadsheet so only the new-to-us people are left in it, then import the rest of the spreadsheet.

**Comment by justdave** [comment]
Another year has passed, and we got the new list from council...  Again, units have been updated (thanks to Mary and Jeremy at SPW) but we don't have contact info updated yet.  The contact info received from council is lacking.  We've determined that we're better off trying to get this contact info from the DEs in each district.  Mary's already gotten the contact list from Eagle Spirit, and Jeremy has likewise gotten the one from Northern Lights, and they will each be working on updating their own chapters.
